packages/polaris-viz/src/components/Grid/tests/__snapshots__/Grid.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Grid /> renders correctly 1`] = `[Function]`;
```

Comment:
I don't think we want this snapshot, do we?

Reply:
we don't!